pandas/conftest.py

```diff
@@ -248,7 +248,19 @@ def tz_aware_fixture(request):
     return request.param
 
 
-@pytest.fixture(params=[str, 'str', 'U'])
+UNSIGNED_INT_DTYPES = ["uint8", "uint16", "uint32", "uint64"]
+SIGNED_INT_DTYPES = [int, "int8", "int16", "int32", "int64"]
```

Comment:
@gfyoung this is ok? it follows our existing pattern?

Reply:
@jreback @gfyoung I only added `COMPLEX_DTYPES`, `STRING_DTYPES` and `ALL_NUMPY_DTYPES` - the rest was just reordering